pkgs/development/python-modules/jinja2/default.nix

```
@@ -3,7 +3,8 @@
 , isPy3k
 , fetchPypi
 , pytest
-, markupsafe }:
+, markupsafe
+, setuptools }:
```

to help with future diffs
```suggestion
, setuptools
}:
```

---

Done